pkgs/applications/networking/browsers/palemoon/default.nix

```
@@ -10,14 +10,14 @@
 
 stdenv.mkDerivation rec {
   name = "palemoon-${version}";
```

`name` gets auto-constructed if you supply `pname` now, best to change that while we're already here.
```suggestion
  pname = "palemoon";
```

---

I see that a lot of other packages use this formula instead:

    pname = "myname";
    name = "${pname}-${version}";

---

> For convenience, you can also use pname and version attributes and mkDerivation will automatically set name to "${pname}-${version}" by default. Since RFC 0035, this is preferred for packages in Nixpkgs, as it allows us to reuse the version easily
> https://github.com/NixOS/nixpkgs/commit/902952d3a3c1f579a6a68da54c4d1403f30ccfdc